<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicSetting</class>
 <widget class="QDialog" name="OBSBasicSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>748</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Basic.Settings.General</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/general.svg</normaloff>:/settings/images/settings/general.svg</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Basic.Settings.Video</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/video.svg</normaloff>:/settings/images/settings/video.svg</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="settingsPages">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalPage">
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>820</width>
              <height>699</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QWidget" name="generalWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <widget class="QGroupBox" name="groupBox_15">
                  <property name="title">
                   <string>Basic.Settings.General</string>
                  </property>
                  <layout class="QFormLayout" name="formLayout_32">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <property name="labelAlignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Basic.Settings.General.Language</string>
                     </property>
                     <property name="buddy">
                      <cstring>language</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="language"/>
                   </item>
                   <item row="1" column="0">
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>170</width>
                       <height>5</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="1" column="1">
                    <widget class="QCheckBox" name="enableAutoUpdates">
                     <property name="text">
                      <string>Basic.Settings.General.EnableAutoUpdates</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupOutput">
                  <property name="title">
                   <string>Basic.Settings.Output</string>
                  </property>
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <property name="labelAlignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="advOutEncLabel">
                     <property name="text">
                      <string>Basic.Settings.Output.Encoder</string>
                     </property>
                     <property name="buddy">
                      <cstring>advOutEncoder</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="advOutEncoder"/>
                   </item>
                   <item row="1" column="0">
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>170</width>
                       <height>5</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="1" column="1">
                    <widget class="QCheckBox" name="advOutApplyService">
                     <property name="text">
                      <string>Basic.Settings.Output.Adv.ApplyServiceSettings</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="videoPage">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Basic.Settings.Video.BaseResolution</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>baseResolution</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="baseResolution">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Basic.Settings.Video.ScaledResolution</string>
           </property>
           <property name="buddy">
            <cstring>outputResolution</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="outputResolution">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Basic.Settings.Video.DownscaleFilter</string>
           </property>
           <property name="buddy">
            <cstring>downscaleFilter</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="downscaleFilter">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="fpsType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentText">
            <string>Basic.Settings.Video.FPSCommon</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>Basic.Settings.Video.FPSCommon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Basic.Settings.Video.FPSInteger</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Basic.Settings.Video.FPSFraction</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QStackedWidget" name="fpsTypes">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QComboBox" name="fpsCommon">
               <property name="currentText">
                <string notr="true">10</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string notr="true">10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">20</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>24 NTSC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>25 PAL</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">29.97</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">48</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>50 PAL</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">59.94</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">60</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QSpinBox" name="fpsInteger">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>120</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QFormLayout" name="formLayout_4">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QSpinBox" name="fpsNumerator">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="fpsDenominator">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Basic.Settings.Video.Numerator</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Basic.Settings.Video.Denominator</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="videoMsg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="themeID" stdset="0">
            <string>error</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>settingsPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fpsType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>fpsTypes</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
